api/app/weather_models/wind_direction_sample.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import numpy as np
+from datetime import datetime
+from collections import defaultdict
+
+logger = logging.getLogger(__name__)
+
+
+class WindDirectionSamples:
```

Comment:
Should this implement SamplesProto? ie `class WindDirectionSample(SamplesProto)`

Reply:
Actually, I wonder if we even need this class. I think the reason we thought it was needed was to avoid the 'reshaping' that happens in `Sample.np_x()`, but I don't think that re-shaping is actually required by any weather parameter is it?

Reply:
I don't think so, I added this to hedge risk but I think it's unnecessary. I'll remove the reshaping from `Samples` and reuse it.